profiles/kentik_snmp/cisco/cisco-wlc.yml

```diff
@@ -182,6 +182,17 @@ metrics:
           name: cLWlanSsid
           OID: 1.3.6.1.4.1.9.9.512.1.1.1.1.4
         tag: wlan_ssid
+      # This object represents average value of peer controller reachable latency in micro seconds
+      - column:
+          name:  cLHaAvgPeerReachLatency
+          OID: 1.3.6.1.4.1.9.9.843.1.5.1
+        tag: Ha_avg_peer_reach_latency
+      #  This object represents current status of bulk sync
+      - column:
+          name:  cLHaBulkSyncStatus
+          OID: 1.3.6.1.4.1.9.9.843.1.5.3
+        tag:  Ha_bulk_sync_status
+
```

Comment:
these are both scalar OIDs from the [cisco-lwapp-ha-mib](https://github.com/netdisco/netdisco-mibs/blob/0e65e5f33694d7cfa818b8c26e100fb63bb2e7ae/cisco/CISCO-LWAPP-HA-MIB.my#L962-L991) and need to be setup outside of the parent object you have them in now.

Reply:
Done